assets/src/components/HorizontalBar.js

```diff
@@ -1,9 +1,16 @@
 import createHorizontalBar from './d3/createHorizontalBar'
-import createChartComponent from './createChartComponent'
-import compose from '../util/compose'
+import { useState } from 'react'
+import useCreateChart from '../hooks/useCreateChart'
+import React from 'react'
```

Comment:
Lines 2 and 4 should be combined into 
`import React, { useState } from 'react`

Reply:
Done